src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -129,6 +128,28 @@ Note: Changes require a restart of PowerToys Run.
 If you enable or don't configure this policy, the user takes control over the enabled state of the Enable paste with AI Advanced Paste setting.
 
 If you disable this policy, the user won't be able to enable Enable paste with AI Advanced Paste setting and use Advanced Paste AI prompt nor set up the Open AI key in PowerToys Settings.
+</string>
+      <string id="ConfigureEnabledFileExplorerAddOnDescription">This policy configures the enabled state for a File Explorer add-on.
+
+If you enable this setting, the add-on will be always enabled and the user won't be able to disable it.
+
+If you disable this setting, the add-on will be always disabled and the user won't be able to enable it.
+
+If you don't configure this setting, users are able to enable or disable the add-on.
+
+The policy "File Explorer add-ons: Configure enabled state" will override this policy.
```

Comment:
```suggestion
Note: If you disable the policy "File Explorer add-ons: Configure enabled state" all File Explorer Add-ons are disabled.
```

@acekirkpatrick 
Am I correct that it only overrides if the File Explorer Add-ons Module is disabled. Because if it is enabled, each add-on can be configured individually.

Reply:
Yes, good catch.

Reply:
You need to commit this line too. Not only line 152.